content/en/docs/1.get-started/1.installation.md

```diff
@@ -18,7 +18,7 @@ Here, you will find information on setting up and running a Nuxt.js project in 4
 
 You can play with Nuxt online directly on CodeSandbox or StackBlitz:
 
-:button-link[Play on CodeSandbox]{href=https://codesandbox.io/s/github/nuxt/codesandbox-nuxt/tree/master/ blank}
+:button-link[Play on CodeSandbox]{.mb-2 .sm:mb-0 href=https://codesandbox.io/s/github/nuxt/codesandbox-nuxt/tree/master/ blank}
```

Comment:
Idealy, thoses `.mb-2 .sm:mb-0` classes are better inside the component, so we avoid propagating it all over the md files. But if we cannot do anything about it here, it's fine.